siliconcompiler/schema/schema_cfg.py

```diff
@@ -84,6 +86,9 @@ def scparam(cfg,
         cfg['help'] = schelp
         cfg['signature'] = signature
         cfg['notes'] = notes
+        # none, optional, mandatory
+        cfg['pernode'] = pernode
+        cfg['pernode_value'] = {}
```

Comment:
Can we make pernode_value --> 'nodevalue'.
We have gone this far without an underscore in the fields. Feels out of place in terms of "lock and feel.", it's an outlier compared to all others. (we have 13 fields so far)

Reply:
Sure, easy change! I'm happy with that.